src/services/user/hooks/userService.js

```diff
@@ -573,6 +573,16 @@ const includeOnlySchoolRoles = async (context) => {
 	return context;
 };
 
+const adminUsersCheckUniqueAccount = async (hook) => {
+	const userService = hook.app.service('/users');
+	const { email } = hook.data;
+	const accounts = await userService.find({ query: { email: email.toLowerCase() } });
+	if (accounts.data.length > 0) {
+		return Promise.reject(new BadRequest('Email already exists.'));
+	}
```

Comment:
Please add this to the service. This is business logic. We should try to keep hooks unit-testable and re-usable.

Instead of `accounts.data.length`, I would suggest using `accounts.total`